cspell.json

```diff
@@ -30,6 +30,8 @@
 		"isbot",
 		"Jérémie",
 		"knip",
+		"Mendiola",
```

Comment:
Hi hi @xav-ie! Could you add his name to https://github.com/SquiggleTools/boston-ts-website instead?

Reply:
Done. Ty